infrastructure-playbooks/purge-cluster.yml

```diff
@@ -531,6 +506,11 @@
       state: absent
     when: ansible_os_family == 'RedHat'
 
+  - name: check for anything running ceph
+    shell: "ps awux | grep -- /usr/bin/[c]eph-"
```

Comment:
I tend to believe simply using `ps -u ceph -U ceph` should be more than enough and less error prone than this `grep`. Another benefit would be to use the `command` module.